tutorials/haskell/gui-application/tutorial.md

```diff
@@ -0,0 +1,825 @@
+---
+title: Creating a GUI application in Haskell
+published: 2016-06-29
+update: 2016-06-29
+ghc: 7.10.3
+lts: 5.18
+libraries: gtk3-0.14.4 glib-0.13.2.2
+language: haskell
+author: Mark
+author-name: Mark Karpov
+---
+
+This tutorial shows how to build a GUI application in Haskell using bindings
+to GTK+. While working on a calculator program we'll cover he following
+topics:
```

Comment:
@mrkkrp Can you add the words "graphical user interface" somewhere? Maybe "graphical user interface (GUI) the first time and then "GUI" for the rest? Also, s/"he following topics"/"the following topics".